apps/dashboard/config/initializers/dev_xdmod.rb

```diff
@@ -0,0 +1 @@
+ENV['XDMOD'] = ""
```

Comment:
This looks like an accidental addition and should be removed

Reply:
Yea you'll likely have to refresh the page because I remove that a bit ago in https://github.com/OSC/ondemand/pull/1009/commits/deff606370d0999b72f1fcd632e448e0c89cbe98.